src/wallet.js

```diff
@@ -71,7 +71,9 @@ MyWallet.getSocketOnMessage = function (message, lastOnChange) {
     if (lastOnChange.checksum !== newChecksum && oldChecksum !== newChecksum) {
       lastOnChange.checksum = newChecksum;
 
-      MyWallet.getWallet();
+      MyWallet.getWallet(function () {
+        WalletStore.sendEvent('on_change');
```

Comment:
Yup, because if any part of `getWallet()` fails, then `MyWallet.wallet = new Wallet(obj)` is never executed, so there's no change to notify the frontend about.